packages/dds/sequence/src/test/intervalRebasing.spec.ts

```diff
@@ -477,11 +489,8 @@ describe("interval rebasing", () => {
 		await assertConsistent(clients);
 	});
 
-	// todo: potentially related to AB#7050
-	//
-	// this is a reduced fuzz test from the suite
-	// `SharedString with rebasing and reconnect`
-	it.skip("...", async () => {
+	it("Can resubmit insert op at the end of the string", async () => {
```

Comment:
the `beforeAll` pattern this suite used does not mix with what this test is trying to do (apply ops to a client before it attaches). This is the justification for refactoring to use an initialization helper function over implicitly doing it in beforeAll in all the tests.